tools/SetupFlow/DevHome.SetupFlow/ViewModels/QuickstartPlaygroundViewModel.cs

```diff
@@ -604,6 +610,17 @@ public void OnQuickstartSelectionChanged()
             ConfigureForProviderSelection();
         }
     }
+
+    [RelayCommand]
+    private void OnLoaded()
+    {
+        var isEnabled = _experimentationService.ExperimentalFeatures.FirstOrDefault(f => string.Equals(f.Id, "QuickstartPlayground", StringComparison.Ordinal))!.IsEnabled;

```

Comment:
should this just instead call https://github.com/microsoft/devhome/blob/92c6ebad3f25f1af24dbdb778826f09674728224/src/Services/ExperimentationService.cs#L28C17-L28C33

Reply:
I think this pattern probably came from logic in https://github.com/microsoft/devhome/blob/92c6ebad3f25f1af24dbdb778826f09674728224/tools/SetupFlow/DevHome.SetupFlow/ViewModels/MainPageViewModel.cs that predated the helpers in ExperimentationService.

Please consider fixing the occurrences there too if you can. :)

Reply:
Ah, good catch. I've modified the code so it now uses ExperimentationService's IsFeatureEnabled(). I've also modified two calls in MainPageViewModel to use IsFeatureEnabled() as well.